File: 백준/Bronze/5622. 다이얼/다이얼.c
#include <stdio.h>
#include <string.h>

int main(void) {
	char arr[16];
	scanf("%s", arr);
	int time = 0;

	for (int i = 0; i < strlen(arr); i++) {
		char alpha = arr[i];
		if (alpha >= 'A' && alpha <= 'C') time += 3;
		else if (alpha >= 'D' && alpha <= 'F') time += 4;
		else if (alpha >= 'G' && alpha <= 'I') time += 5;
		else if (alpha >= 'J' && alpha <= 'L') time += 6;
		else if (alpha >= 'M' && alpha <= 'O') time += 7;
		else if (alpha >= 'P' && alpha <= 'S') time += 8;
		else if (alpha >= 'T' && alpha <= 'V') time += 9;
		else if (alpha >= 'W' && alpha <= 'Z') time += 10;
	}
	printf("%d", time);

	return 0;
}

File: 백준/Bronze/5622. 다이얼/다이얼.py
arr = list(input())[:15]
length = len(arr)
time = 0
for i in range(length):
    alpha = arr[i]
    if (alpha >= 'A' and alpha <= 'C'): time += 3
    elif (alpha >= 'D' and alpha <= 'F'): time += 4
    elif (alpha >= 'G' and alpha <= 'I'): time += 5
    elif (alpha >= 'J' and alpha <= 'L'): time += 6
    elif (alpha >= 'M' and alpha <= 'O'): time += 7
    elif (alpha >= 'P' and alpha <= 'S'): time += 8
    elif (alpha >= 'T' and alpha <= 'V'): time += 9
    elif (alpha >= 'W' and alpha <= 'Z'): time += 10

print(time)